sdk/eventhub/event-hubs/changelog.md

```diff
@@ -1,87 +1,101 @@
+### 2019-10-11 2.1.2
+
+- Errors thrown in the user-provided message handler to `client.receive` are now properly caught by the client.
```

Comment:
EventHubReceiver.receive?

Reply:
No, in Track 1, the receive method was on the client not the receiver. The receiver was not exposed to the user